app/models/page.rb

```diff
@@ -530,7 +529,7 @@ def clear_links(text_type)
     if self.page_article_links.present?
       self.clear_article_graphs
       # clear out the existing links to this page
-      PageArticleLink.where("page_id = #{self.id} and text_type = '#{text_type}'").delete_all
+      PageArticleLink.where("page_id = #{self.id} and text_type = '#{text_type}'").destroy_all
```

Comment:
Doing delete_all does not trigger active_record callbacks , therefore no validations.

Moving forward, unless we specifically want to avoid callbacks, we should use destroy instead of delete in order to avoid unvalidated changes.